### PR TITLE
EDSC-3029: Display data on the timeline

### PR DESCRIPTION
**Pull request by trevorlang**
Displays data on the timeline for 1 or more collections, limited to 3.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,11 +21,13 @@ import { roundTime } from './utils/roundTime'
 
 import {
   INTERVAL_BUFFER,
+  MAX_DATA_ROWS,
   MAX_INTERVAL_BUFFER,
   TEMPORAL_SELECTION_HEIGHT
 } from './constants'
 
 import './index.scss'
+import { TimelinePrimarySection } from './components/TimelinePrimarySection/TimelinePrimarySection'
```
Comment (macrouch) [inline src/index.js]: Move import to other component imports

**Review by macrouch — COMMENTED** [review]
I saw this missing prop error when running the tests

``` 
PASS  src/__tests__/index.test.js (7.044 s)
  ● Console

    console.error
      Warning: Failed prop type: The prop `timelineWrapperWidth` is marked as required in `TimelineDataSection`, but its value is `undefined`.
          in TimelineDataSection (created by ForwardRef)
```

File: src/index.js [hunk 2]
```diff
@@ -788,7 +838,7 @@ export const EDSCTimeline = ({
 
   useEffect(() => {
     window.addEventListener('mouseup', onWindowMouseUp)
-    window.addEventListener('mousemove', onWindowMouseMove)
+    window.addEventListener('mousemove', onWindowMouseMove, false)
```
Comment (trevorlang) [inline src/index.js]: Remove `false`

File: src/index.js [hunk 3]
```diff
@@ -763,6 +812,7 @@ export const EDSCTimeline = ({
    */
   const onWindowMouseMove = (e) => {
     if (dragging) onTimelineDrag(e)
+    // if (dragging) requestAnimationFrame(() => onTimelineDrag(e))
```
Comment (abbottry) [inline src/index.js]: debug?

File: src/index.js [hunk 4]
```diff
@@ -908,6 +964,18 @@ EDSCTimeline.defaultProps = {
 
 EDSCTimeline.propTypes = {
   center: PropTypes.number,
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string, // ?
```
Comment (abbottry) [inline src/index.js]: unsure of why this is hear or a stagnant comment?

**Review by abbottry — APPROVED** [review]

**Review by macrouch — APPROVED** [review]

**Pull request merged**